src/Umbraco.Tests.Integration/Umbraco.Infrastructure/Services/ContentServiceEventTests.cs

```diff
@@ -34,7 +34,7 @@ public void SetupTest()
             ContentRepositoryBase.ThrowOnWarning = true;
             _globalSettings = new GlobalSettings();
             // TODO: remove this once IPublishedSnapShotService has been implemented with nucache.
-            Core.Services.Implement.ContentTypeService.ClearScopeEvents();
+            global::Umbraco.Core.Services.Implement.ContentTypeService.ClearScopeEvents();
```

Comment:
I don't think this would work unfortunately.  Reason being here we're looking to call a static method on `ContentTypeService`, not an instance one that would be available from the private field.  I've had to introduce the `global::` prefix to disambiguate the reference due to `Umbraco` (which without this it picks up from the `Umbraco.Tests.Integration.Umbraco` part of the test class's namepace.